Client/Frontend/Theme/ThemedTableViewController.swift

```diff
@@ -96,15 +90,19 @@ class ThemedHeaderFooterViewBordersHelper: ThemeApplicable {
         view.addSubview(topBorder)
         view.addSubview(bottomBorder)
 
-        topBorder.snp.makeConstraints { make in
-            make.left.right.top.equalTo(view)
-            make.height.equalTo(0.25)
-        }
-
-        bottomBorder.snp.makeConstraints { make in
-            make.left.right.bottom.equalTo(view)
-            make.height.equalTo(0.5)
-        }
+        topBorder.translatesAutoresizingMaskIntoConstraints = false
+        bottomBorder.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
This line too.

Reply:
Please see [9199202](https://github.com/mozilla-mobile/firefox-ios/pull/17090/commits/9199202616725913546eb10c20614145c6ea71ae)